packages/ui-components/src/components/Chart/Table/getChartTableData.js

```diff
@@ -90,8 +90,10 @@ const processData = viewContent => {
 export const getChartTableData = viewContent => {
   const columns = useMemo(() => processColumns(viewContent), [JSON.stringify(viewContent)]);
   const data = useMemo(() => processData(viewContent), [JSON.stringify(viewContent)]);
+  const { chartConfig } = viewContent;
```

Comment:
I played around within this file and found we actually can add the label in here specifying the `Header` config as a function in  `processColumns()`. API reference: https://react-table.tanstack.com/docs/api/useTable

I feel it would be better to put the logic here along with other logics. What I did is to put the following code after line 64:

>   const configColumns = Object.keys(chartDataConfig).map(columnKey => {
    return {
      Header: ({ column }) => {
        return chartConfigs[column.id]?.label || column.id;
      },
      id: columnKey, 


Happy to have a chat to talk it through 😬.